tests/components/geo_location/test_trigger.py

```diff
@@ -118,18 +118,20 @@ async def test_if_fires_on_zone_enter(
         blocking=True,
     )
 
+    assert len(service_calls) == 2
+
```

Comment:
`service_calls` fixture now includes the call to `automation.turn_off`
So call count increased by 1, but then `async_set` doesn't increase it a second time.